src/com/google/enterprise/adaptor/opentext/OpentextAdaptor.java

```diff
@@ -500,6 +502,10 @@ public void init(AdaptorContext context) {
       this.localNamespace =
           getLocalNamespace(this.globalNamespace, this.contentServerUrl);
       log.log(Level.CONFIG, "local namespace: {0}", this.localNamespace);
+      this.pushLocalGroupsOnly = Boolean.parseBoolean(
```

Comment:
In the code just above, you log the value before you parse it. I think we should be consistent either way, and I mildly prefer the other approach.